WinUIGallery/App.xaml.cs

```diff
@@ -60,6 +61,23 @@ protected override void OnLaunched(Microsoft.UI.Xaml.LaunchActivatedEventArgs ar
         MainWindow.Closed += (s, e) =>
         {
             BadgeNotificationManager.Current.ClearBadge();
+            
+            // Close all remaining active windows to prevent resource disposal conflicts
+            var activeWindows = new List<Window>(WindowHelper.ActiveWindows);
+            foreach (var window in activeWindows)
+            {
+                if (window != s) // Don't try to close the window that's already closing
```

Comment:
Since `s` is typed as `object`, using `Equals` avoids type mismatch warnings or squiggles.

```cs
if (!window.Equals(s))
```

Reply:
Resolved